scale-test/fioload-template.yaml

```diff
@@ -0,0 +1,47 @@
+---
```

Comment:
This only ever does read. By default, cstor is configured with 4GB iirc, so let's say its 3GB of ARC. Most IO will be in memory once the file is laid out. 

Also, the iodepth of 2 is not a realistic value, it should match the value that the target supports.